typings.d.ts

```diff
@@ -22,8 +22,8 @@ declare namespace betterAjvErrors {
     (
       schema: any,
       data: any,
-      errors: ErrorObject[],
-      options: betterAjvErrors.IInputOptions
+      errors?: ErrorObject[] | null,
```

Comment:
Is the `null` necessary?

Reply:
[Yep!](https://github.com/epoberezkin/ajv/blob/8e7f47a9f29fc4ba487720c301142d8706a38a8a/lib/ajv.d.ts#L119) That's what made me make this PR, IIRC.

Reply:
I don't follow, why allow `undefined`, but not `null`? In the link I provided, `ajv`'s `errors` property on the validation function may be `null`.